gatekeeperv2.kvp

```diff
@@ -0,0 +1,86 @@
+Meta.DisplayName=GatekeeperV2 Bot
+Meta.Description=AMP Discord Bot created by k8thekat - https://github.com/k8thekat/GatekeeperV2
+Meta.OS=Linux
+Meta.Arch=x86_64
+Meta.Author=k8thekat, IceOfWraith
+Meta.URL=https://github.com/k8thekat/GatekeeperV2
+Meta.DisplayImageSource=url:https://avatars.githubusercontent.com/u/68672235?v=4
+Meta.EndpointURIFormat=steam://connect/{0}
+Meta.ConfigManifest=gatekeeperv2config.json
+Meta.MetaConfigManifest=gatekeeperv2metaconfig.json
+Meta.ConfigRoot=gatekeeperv2.kvp
+Meta.MinAMPVersion=
+Meta.SpecificDockerImage=cool11989/iceofwraith:gatekeeperv2
+Meta.DockerRequired=False
+Meta.ContainerPolicy=Recommended
+Meta.Prerequsites=[]
+Meta.ConfigReleaseState=NotSpecified
+Meta.NoCommercialUsage=False
+App.DisplayName=GatekeeperV2 Bot
+App.RootDir=./gatekeeperv2/
+App.BaseDirectory=./gatekeeperv2/
+App.ExecutableWin=$PATH/python3
+App.ExecutableLinux=/usr/bin/python3
+App.WorkingDir=GatekeeperV2-main
+App.LinuxCommandLineArgs=
+App.WindowsCommandLineArgs=
+App.CommandLineArgs=start.py -token {{dev}}{{debug}}{{discord}}{{super}} {{guildID}}
+App.UseLinuxIOREDIR=False
+App.AppSettings={}
+App.EnvironmentVariables={}
+App.CommandLineParameterFormat=+{0} {1}
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
+App.ExitString=stop
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.Ports=[{"Protocol":"UDP","Port":0,"Ref":"Port","Name":"Port","Description":"Dummy port"}]
```

Comment:
My experience with the original configuration of Core Keeper shows this can cause issues with AMP. Some have been fixed, but I am not sure all (for example, iptables errors relating to Docker when trying to reactivate an instance).

Reply:
That's the config I used for an example. Do you suggest I put in an actual port as the dummy?

Reply:
Yes, for an easier life

Reply:
Dummy port is now a dummy port.